tests/sonyci/asset.toml

```diff
@@ -0,0 +1,2 @@
+[sonyci]
+asset = '554544ceaf6b4c94a4a06cee5bc1f39f'
```

Comment:
is this used? Seems redundant with the `asset_id` fixture

Reply:
It's a real asset in the `My Workspace` test workspace:

`cpb-aacip-e4308199588.txt`

> hello people!!!

Reply:
right, but we can get rid of the toml file right, and just use the fixture. That's all i can see being used.